packages/components/src/ToolTip.tsx

```diff
@@ -1,128 +1,104 @@
 import * as React from "react";
 import * as ReactDOM from "react-dom";
 import styled from "styled-components";
-import { colors } from "./styleConstants";
+import { colors, fonts } from "./styleConstants";
 
 export interface ToolTipProps {
   explainerText: React.ReactNode;
   disabled?: boolean;
+  positionBottom?: boolean;
 }
 
 export interface ToolTipState {
   open: boolean;
 }
 
 export interface TipProps {
-  top: number;
-  left: number;
+  positionBottom?: boolean;
 }
 
-const TipContainer = styled<TipProps, "div">("div")`
+const TipContainer = styled.div`
   position: absolute;
-  top: ${(props: TipProps) => props.top}px;
-  left: ${(props: TipProps) => props.left}px;
-  height: 0px;
+  top: 0;
```

Comment:
These weren't props that were passed in they were calculated based on page position but I made the tip relative to the marker instead of the page so they are no longer needed.

Reply:
👍

Reply:
These changes did actually break tooltips in the plugin! WordPress renders our various components in siloed spots for WP plugins (e.g. inside a nav bar) that have overflow hidden, so with the tooltips relative to the target they get cut off.

For that reason (as well as not having to worry about z-indexes) I think absolutely positioned on <body> is probably the way to go. @sruddy do you remember why you made this change so I can make sure things work when I change them back, or maybe we need to make absolute vs. relative a flag that gets passed in?

Reply:
No worries haha. Yeah that sounds a bit gnarly, might need to do something like jquery's `offsetParent`. I'm going to go ahead and give absolutely positioned a shot and just test the 26 tooltips we're currently have in the codebase to make sure they're all good and try to fix any bugs that crop up!